### PR TITLE
CNV-19224: Hypervisor logs RN

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.11

Issue: [CNV-19224](https://issues.redhat.com/browse/CNV-19224)

Link to docs preview: [Release notes](http://file.tlv.redhat.com/apinnick/CNV-19224-rn-verbose-logging/virt/virt-4-11-release-notes.html#virt-4-11-new)

Additional information:

### CODE REVIEW

**Comment by apinnick** [comment]
@stu-gott @fabiand 
Can you review this RN PR?

File: virt/virt-4-11-release-notes.adoc [hunk 1]
```diff
@@ -50,6 +51,7 @@ The SVVP Certification applies to:
 //CNV-18162 Move MTV action from VM list page
 
 //CNV-19224 Hypervisor logs
+* You can configure the verbosity level of the `virtLauncher`, `virtHandler`, `virtController`, `virtAPI`, and `virtOperator` pod logs to debug specific components by editing the `HCO` object.
```
Comment (stu-gott) [inline virt/virt-4-11-release-notes.adoc:54]: Suggest a rewording:

You can configure the verbosity level of the `virtLauncher`, `virtHandler`, `virtController`, `virtAPI`, and `virtOperator` pod logs to debug specific components by editing the `HyperConverged` custom resource (CR).

**Review by stu-gott — COMMENTED** [review]
One small nomenclature change

File: virt/virt-4-11-release-notes.adoc [hunk 2]
```diff
@@ -50,6 +51,7 @@ The SVVP Certification applies to:
 //CNV-18162 Move MTV action from VM list page
 
 //CNV-19224 Hypervisor logs
+* You can configure the verbosity level of the `virtLauncher`, `virtHandler`, `virtController`, `virtAPI`, and `virtOperator` pod logs to debug specific components by editing the `HCO` custom resource (CR).
```
Comment (sjhala-ccs) [inline virt/virt-4-11-release-notes.adoc:54]: s/`HCO`/`HyperConverged`

**Review by sjhala-ccs — COMMENTED** [review]
Just one nit, otherwise LGTM

**Pull request merged**